tests/unity_tests/pam_argv_parse_test.c

```diff
@@ -65,7 +65,6 @@ static void test_pam_argv_conf() {
 
 int main() {
     UNITY_BEGIN();
```

Comment:
```suggestion
    UNITY_BEGIN();
    RUN_TEST(test_pam_config_NULL);
```
to put the test back